handlers/hormuud/hormuud.go

```diff
@@ -0,0 +1,202 @@
+package hormuud
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/buger/jsonparser"
+	"github.com/garyburd/redigo/redis"
+	"github.com/nyaruka/courier"
+	"github.com/nyaruka/courier/handlers"
+	"github.com/nyaruka/courier/utils"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	maxMsgLength = 160
+	tokenURL     = "https:// smsapi.hormuud.com/token"
```

Comment:
space?

Reply:
Their docs have those and I copy pasted but ya that's wrong, fixed.